src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb

```diff
@@ -2356,23 +2356,19 @@ End Module]]>,
 
     <Fact()>
     Public Sub ParseWarningDirective_LineContinuation2()
```

Comment:
Please duplicate this test instead. One variant should use LangVersion 15.3 (and produce an error), and the other should use 15.5 (marked with PROTOTYPE, and produce no error).

Reply:
How do I mark with PROTOTYPE, I have the updates locally including duplicate tests.